phpBB/styles/prosilver/template/posting_body.html

```diff
@@ -1,4 +1,7 @@
 
+<!-- INCLUDEJS jquery.caret.js -->
```

Comment:
You are now including this for all use cases of posting_body, but this is also used for things like sending a private message or editing a post. It might be a good idea to limit when you want to include this file.